### PR TITLE
Docs: Added "manageAlerts" provisioning option.

### PR DESCRIPTION
**Pull request by BitProcessor**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by achatterjee-grafana** [comment]
Looks good, will be good to have a dev review.

**Review by achatterjee-grafana — APPROVED** [review]

File: docs/sources/administration/provisioning.md [hunk 1]
```diff
@@ -154,6 +154,7 @@ Since not all datasources have the same configuration settings we only have the
 | maxSeries               | number  | Influxdb                                                         | Max number of series/tables that Grafana processes                                                                                                |
 | httpMethod              | string  | Prometheus                                                       | HTTP Method. 'GET', 'POST', defaults to POST                                                                                                      |
 | customQueryParameters   | string  | Prometheus                                                       | Query parameters to add, as a URL-encoded string.                                                                                                 |
+| manageAlerts            | boolean | Prometheus                                                       | Manage alerts via Alerting UI                                                                                                      |
```
Comment (domasx2) [inline docs/sources/administration/provisioning.md:157]:
```suggestion
| manageAlerts            | boolean | Prometheus and Loki                                                  | Manage alerts via Alerting UI                                                                                                      |
```

**Comment by BitProcessor** [comment]
@achatterjee-grafana  not sure why the license/cla check is pending ? 🤔

**Comment by achatterjee-grafana** [comment]
> @achatterjee-grafana not sure why the license/cla check is pending ? 🤔

This is an unfortunate but random failure. Do you mind reverting your commit and re-push the change again? That will kickstart the checks. Or we can do it for you. Thanks!

**Comment by BitProcessor** [comment]
@achatterjee-grafana was about to do it, but noticed you already did it in my place 🥳 
Thanks for that.

**Comment by achatterjee-grafana** [comment]
Thank you, @BitProcessor for the contribution!

**Pull request merged**